src/ACF/Image.php

```diff
@@ -0,0 +1,61 @@
+<?php # -*- coding: utf-8 -*-
+
+namespace MultisiteGlobalMedia\ACF;
+
+use MultisiteGlobalMedia\Helper;
+use MultisiteGlobalMedia\SingleSwitcher;
+use MultisiteGlobalMedia\Site;
+
+class Image
+{
+
+    use Helper;
+
+    /**
+     * @var Site
+     */
+    private $site;
+
+    /**
+     * @var SingleSwitcher
+     */
+    private $siteSwitcher;
+
+    /**
+     * @var \ACF_Data
+     */
+    private $store;
+
+    /**
+     * Image constructor
+     *
+     * @param Site $site
+     * @param SingleSwitcher $siteSwitcher
+     */
+    public function __construct(Site $site, SingleSwitcher $siteSwitcher)
+    {
+        $this->site = $site;
+        $this->siteSwitcher = $siteSwitcher;
+        $this->store = acf_get_store('values');
```

Comment:
Could the returned value of `acf_get_store` function get injected as dependency instead of calling the function within the constructor?

Reply:
Thanks for reviewing my code!

I've refactored the code to (hopefully) improve the bootstrapping process and testabilty. However I don't think that `acf_get_store` should be injected as dependency now. This function can be mocked for testing purposes.

Reply:
You don't need to mock if you inject the values :)

Reply:
Yes, that's true, but in that case `Plugin.php` needs to check if ACF exists/is enabled. I've rewritten that part so `Plugin.php`calls `ACF\Image::bootstrap($site, $singleSwitcher)` without further knowledge what else is required.

Reply:
I do not agree with the classic WP approach to make classes aware of more then one concern. Now the class is addressing two concerns, one is Bootstrapping which is responsible of the Plugin class and the other one is the main concern which is adding Image support for ACF.

Please see my comment to the changes.

Reply:
I see your point and I've reverted the responsibility of bootstrapping back to `Plugin.php`.